src/vectorsynchronizer.js

```diff
@@ -81,6 +81,22 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
     csPrimitives.show = olLayer.getVisible();
   });
 
+  var onOpacityChange = function() {
```

Comment:
It does not work without this code. Can you point me to the place where point features are destroyed and re-created upon layer change? Also, where are you envisioning more detailed ol3 change events? You can already listen for specific property changes on the layer, which is done here.